app/components/support_interface/reasons_for_rejection_dashboard_component.rb

```diff
@@ -18,11 +20,12 @@ def total_rejection_count(reason)
 
     def percentage_rejected_for_reason(reason)
       count = current_month_rejection_count(reason) + previous_rejection_count(reason)
-      total = @rejection_reasons.inject(0) { |sum, hash| sum + hash['count'] }
-      formatted_percentage(count, total)
+      formatted_percentage(count, total_structured_rejection_reasons_count)
```

Comment:
Everything looks good - only a minor question - can the `total_rejection_count` method be passed here in place of count?

Reply:
💯 good spot